stl/inc/filesystem

```diff
@@ -2134,12 +2134,14 @@ namespace filesystem {
         directory_entry(const directory_entry&)     = default;
         directory_entry(directory_entry&&) noexcept = default;
         explicit directory_entry(const filesystem::path& _Path_arg) : _Cached_data{}, _Path(_Path_arg) {
-            (void) _Refresh(_Cached_data, _Path);
+            refresh();

```

Comment:
This reverts a very intentional change, see discussion in the original issue report.